litle.yml

```diff
@@ -1,15 +1,15 @@
 :litle:
   - :account_id: "USD"
     :merchant_id: "your-merchant-id-USD"
-    :username: "your-username"
+    :login: "your-username"
     :password: "your-password"
-    :secure_page_url: "https://request-prelive.np-securepaypage-litle.com"
```

Comment:
You need to keep this so we can still test PayPage locally (`rackup -Ilib -E test`)